pages/index.js

```diff
@@ -1,31 +1,155 @@
-import ComingSoon from '../components/styles/ComingSoon.styled';
 import styled from 'styled-components';
-import ComingSoonImage from '../components/ComingSoonImage';
+import { motion } from 'framer-motion';
 import MotionHeader from '../components/styles/MotionHeader.styled';
 import MotionMain from '../components/styles/MotionMain.styled';
 import MotionFooter from '../components/Footer';
+import StyledSection from '../components/styles/StyledSection.styled';
+import SectionContainer from '../components/styles/SectionContainer.styled';
+import Image from 'next/image';
+import SoCBanner from '../images/seed-of-cain/banner.jpg';
+import SoCLogo from '../images/seed-of-cain/logo.png';
+import {
+	RobotIconHexagon,
+	BlockchainIconHexagon,
+	MicrochipIconHexagon,
+} from '../components/HexagonIcons';
+import LogoWithText from '../components/LogoWithText';
```

Comment:
is this supposed to be `LogoTextImage`?

Reply:
I forgot to include something in my commit, I'll get it added.